pipelines/release/nightly_release.groovy

```diff
@@ -143,51 +143,99 @@ notify.wrap {
     def triggerMe = [:]
 
     triggerMe['build Science Platform Notebook Aspect Lab image'] = {
-      retry(retries) {
-        // based on lsstsqre/stack image
-        build(
-          job: 'sqre/infra/build-sciplatlab',
-          parameters: [
-            string(name: 'TAG', value: eupsTag),
-            booleanParam(name: 'NO_PUSH', value: false),
-            booleanParam(name: 'JLBLEED', value: false),
-            string(
-              name: 'IMAGE_NAME',
-              value: scipipe.release.step.build_sciplatlab.image_name,
-            ),
-            // BASE_IMAGE is the registry repo name *only* without a tag
-            string(
-              name: 'BASE_IMAGE',
-              value: stackResults.docker_registry.repo,
-            ),
-          ],
-          wait: false,
-        )
-      } // retry
+      stages {
```

Comment:
`stages` is a "declarative" pipeline construct which I don't think will work under "scripted" piepline.